### ISSUE TITLE
Create the blog component for the start page

### ISSUE THREAD

**Issue by majarzabek** [michalusio/KuneKune#12]
It should look as follow:

![image](https://github.com/michalusio/KuneKune/assets/48184243/2c2fab55-eacb-4bae-a6ad-0c17c3fa2a6d)

mobile version is a bit different:

![image](https://github.com/michalusio/KuneKune/assets/48184243/c35b5f3e-dc8c-4cad-b388-b28315a56de7)

helpful css' concepts:

https://www.w3schools.com/css/css3_object-fit.asp
https://css-tricks.com/snippets/css/a-guide-to-flexbox/

**Comment by michalusio** [comment]
> [!NOTE]
> It will be useful to compartmentalize - first create a BlogItem (picture + text + button), then combine two items into a BlogComponent

**Comment by majarzabek** [comment]
Yes, also to achieve this view you will have to mock the data. It will look something like this:
```
<h1>Our blog</h1>
<h2>Introduction text bla bla</h2>
<BlogSubComponent imageUrl={something} title={something} heading={something} articleUrl={something}/>
<BlogSubComponent imageUrl={something} title={something} heading={something} articleUrl={something}/>
```
if we had real data, it would look differently:
```
<h1>Our blog</h1>
<h2>Introduction text bla bla</h2>
{articleList.length>0 && articleList.length.map((article: BlogArticle) => (
<BlogSubComponent key={article.url} imageUrl={article.imageUrl} title={article.title} heading={article.heading} articleUrl={article.url}/>
)) }
```
Micha will tell you more about Types and keys :D

**Comment by Gyllenberg** [comment]
This will probably take me way to long to achieve, but imma start it
